packages/astro/src/runtime/client/dev-toolbar/apps/audit/index.ts

```diff
@@ -111,6 +116,119 @@ export default {
 						},
 					})
 				);
+
+				const auditListWindow = createWindowElement(
+					`
+					<style>
+						astro-dev-toolbar-window {
+  	          left: initial;
+              top: 8px;
+              right: 8px;
+              transform: none;
+              width: 320px;
+              max-height: 320px;
+              padding: 0;
+              overflow: hidden;
+						}
+
+						hr {
+						  margin: 0;
+						}
+
+						header {
+						  display: flex;
+							justify-content: space-between;
+							align-items: center;
+						  padding: 18px;
+						}
+
+						h1 {
+						  font-size: 22px;
+  						font-weight: 600;
+  						color: #fff;
+						}
+
+						ul, li {
+						  margin: 0;
+							padding: 0;
+							list-style: none;
+						}
+
+						h1, h2 {
+						  margin: 0;
+						}
+
+						h3 {
+      		    margin: 0;
+              margin-bottom: 8px;
+              color: white;
+              white-space: nowrap;
+						}
+
+						.audit-title {
+						  font-weight: bold;
+							color: white;
+							margin-right: 1ch;
+						}
+
+						#audit-list {
+  						display: flex;
+              flex-direction: column;
+              overflow: auto;
+						}
+					</style>
+
+					<header>
+					  <h1>Audits</h1>
+						<astro-dev-toolbar-badge size="large">${audits.length} problem${
+							audits.length > 1 ? 's' : ''
+						} found</astro-dev-toolbar-badge>
+					</header>
+					<hr />`
+				);
+
+				const auditListUl = document.createElement('ul');
+				auditListUl.id = 'audit-list';
+				audits.forEach((audit, index) => {
+					const resolvedRule = resolveAuditRule(audit.rule, audit.auditedElement);
+					const card = document.createElement('astro-dev-toolbar-card');
+
+					card.shadowRoot.innerHTML = `
+					<style>
+					 :host>button {
+						  text-align: left;
+							box-shadow: none !important;
+							${
+								index + 1 < audits.length
+									? 'border-radius: 0 !important;'
+									: 'border-radius: 0 0 8px 8px !important;'
+							}
+						}
+
+						:host>button:hover {
+						  cursor: pointer;
+						}
+					</style>`;
+
+					card.clickAction = () => {
+						audit.highlightElement.scrollIntoView();
+						audit.highlightElement.focus();
+					};
+					const h3 = document.createElement('h3');
+					h3.innerText = finder(audit.auditedElement);
```

Comment:
Used a package for this, because it turns out that this is super complicated to do correctly